### ISSUE TITLE
Combobox-readonly: duplicated attributes in both span and input

### ISSUE THREAD

**Issue by cynthiahhc** [eBay/ebayui-core#706]
# Bug Report

## Description
After entering the id as an attribute into `ebay-combobox-readonly`, the id shows up in both `span` and `input` which should only shows up in `input.

```<ebay-textbox
  id:scoped="field"
  name=input.name
  multiline=(input.maxLength > 100)
  fluid=input.fluid
  value=input.value
  on-textbox-change('emit', 'textbox-change')
  on-textbox-input('emit', 'textbox-input')
  on-textbox-focus('emit', 'textbox-focus')
  on-textbox-blur('emit', 'textbox-blur')
  on-textbox-keydown('emit', 'textbox-keydown') />  
```
![Screen Shot 2019-06-11 at 12 23 04](https://user-images.githubusercontent.com/12440585/59301739-2900dd80-8c47-11e9-9fad-e1c9ec32715b.png)

https://github.com/eBay/ebayui-core/blob/master/src/components/ebay-combobox-readonly/template.marko#L4

## Workaround
It should be something similar to `ebay-combobx` https://github.com/eBay/ebayui-core/blob/master/src/components/ebay-combobox/template.marko#L36

**Comment by ianmcburnie** [comment]
Thanks for reporting this!

**Comment by ianmcburnie** [comment]
App team moved over to ebay-listbox, reducing this to P4.

**Comment by ianmcburnie** [comment]
Closing as wont fix, given this is a hidden/deprecated component.
